src/editors/sharedComponents/InsertLinkModal/utils.js

```diff
@@ -0,0 +1,143 @@
+import cloneDeep from 'lodash.clonedeep';
+
+export const blockTypes = {
+  section: 'chapter',
+  subsection: 'sequential',
+  unit: 'vertical',
+};
```

Comment:
Assuming the rest of the moves I suggested in this file happen, this and `formatBlockPath` would be the only things left, at which point it might be reasonable to move each out into their own respective files (this could be in `blockTypes.js`).

That being said, we already have a file defining `blockTypes` in this repo here
https://github.com/openedx/frontend-lib-content-components/blob/f33a3b5521d709ae4b0a98a5bdd9744060177816/src/editors/data/constants/app.js#L4-L11

so it might make sense to come up with a new name for this `blockTypes`.